tutorials/websocket_server.md

```diff
@@ -0,0 +1,91 @@
+\page websocket_server Websocket server for gzweb
+
+The websocket server system starts up a websocket server for the web-based
+Gazebo visualization tool, [gzweb](https://github.com/gazebo-web/gzweb).
+
+## Usage
+
+The websocket server system can be included in an world SDF file inside the
+`<world>` SDF element like other systems.
+
+An simple example is shown below :
+
+
+```xml
```

Comment:
```suggestion
An simple example is shown below :

```xml
```

Reply:
Knit: Grammar. Should be "A single example is shown below" not "An"

Reply:
remove line space and fixed grammar. [f542160](https://github.com/gazebosim/gz-sim/pull/2807/commits/f5421604555bb7a5b990589ea032fd9d1471ecfa)